hail/python/test/hailtop/utils/test_filesize.py

```diff
@@ -0,0 +1,28 @@
+from pytest import raises
+from hailtop.utils.filesize import filesize
+
+def test_filesize():
+    with raises(ValueError):
+        filesize(-1)
+        filesize(-1023)
+        filesize(-1024)
+        filesize(-1025)
+        filesize(-1024**3)
```

Comment:
I think only the first invocation is getting tested here. These need to go in separate `with`s.

Reply:
good catch! fixed!